### DISCUSSION TITLE
Code Table Admin Meeting Reminders

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#6399] [Announcements]
@ArctosDB/arctos-code-table-administrators 

Just a reminder that we have a meeting scheduled for Thursday, June 15 at 1PM Pacific Time. I will miss this meeting as I will be on vacation. I have placed a couple of items on the [Agenda](https://docs.google.com/document/d/1eZ_S8VTp-m5Wrbb6cAK96ODPZpt2-Hvnvt6yfU9qVwg/edit?usp=sharing). 

However, there is a [HUGE backlog of code table requests](https://github.com/orgs/ArctosDB/projects/23/views/1?layout=board). We cannot address code table requests only at monthly meetings. We need to be reviewing these in a more timely fashion, or we need a new process for approving new code table terms. Ideas are appreciated and this has also been added to the agenda.

Sorry for missing this.

Adios,
Teresa

**Comment by Jegelewicz** [comment]
@ArctosDB/arctos-code-table-administrators 

Just a reminder that we have a meeting scheduled for Thursday, July 6 at 1PM Pacific Time. [Agenda](https://docs.google.com/document/d/1eZ_S8VTp-m5Wrbb6cAK96ODPZpt2-Hvnvt6yfU9qVwg/edit?usp=sharing). 

Adios,
Teresa

**Comment by Jegelewicz** [comment]
@ArctosDB/arctos-code-table-administrators I suggest we move this meeting to July 1 at 2PM. Please let me know if you can meet at that time.

**Reply by Nicole-Ridgwell-NMMNHS** [reply]
I'm onboarding someone that day and am not sure if I can be available.

**Comment by Jegelewicz** [comment]
@ArctosDB/arctos-code-table-administrators I never moved the meeting. Will anyone be around tomorrow for Code Tables or should I just cancel?

**Reply by mkoo** [reply]
Cancel for the holiday!
The summer is a bit slow and hard to reschedule so it's ok in my view to skip July
Thanks and happy 4th!
